akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -8,20 +8,58 @@
 
 //#log
 import akka.stream.Attributes;
-
+import akka.stream.javadsl.Source;
 //#log
 
+import java.time.Duration;
+import java.util.Arrays;
+
+
 class SourceOrFlow {
 
   void logExample() {
     Flow.of(String.class)
-      //#log
-      .log("myStream")
-      .addAttributes(Attributes.createLogLevels(
-        Attributes.logLevelOff(), // onElement
-        Attributes.logLevelError(), // onFailure
-        Attributes.logLevelInfo())) // onFinish
+        //#log
+        .log("myStream")
+        .addAttributes(Attributes.createLogLevels(
+            Attributes.logLevelOff(), // onElement
+            Attributes.logLevelError(), // onFailure
+            Attributes.logLevelInfo())) // onFinish
     //#log
     ;
   }
+
+  void conflateExample() {
+    //#conflate
+    Source.cycle(() -> Arrays.asList(1, 10, 100).iterator())
+        .throttle(10, Duration.ofSeconds(1)) // fast upstream
+        .conflate((Integer acc, Integer el) -> acc + el)
+        .throttle(1, Duration.ofSeconds(1)); // slow downstream
+    //#conflate
+  }
+
+  static //#conflateWithSeed-type
+  class NewType {
```

Comment:
Maybe call it `SummedValue` or something that describes what it does rather than this?